configuration.example.yaml

```diff
@@ -75,6 +75,6 @@ influxdb:
   # The token to secure the influxdb endpoint.
   token: SecretBase64Token==
   # The organisation set in your influxdb configuration.
```

Comment:
Let's fix it in the string as well. Otherwise, we might get confused with the American and British versions ...

Reply:
```suggestion
  # The organization set in your influxdb configuration.
```